ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/CardinalClient.java

```diff
@@ -78,27 +78,24 @@ private void configureCardinal(Context context, Configuration configuration,
         cardinalConfigurationParameters.setEnableDFSync(true);
 
         switch (request.getUiType()) {
-            case ThreeDSecureRequest.NATIVE:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.NATIVE);
-            case ThreeDSecureRequest.HTML:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.HTML);
-            case ThreeDSecureRequest.BOTH:
-                cardinalConfigurationParameters.setUiType(CardinalUiType.BOTH);
+            case NATIVE: cardinalConfigurationParameters.setUiType(CardinalUiType.NATIVE);
+            case HTML: cardinalConfigurationParameters.setUiType(CardinalUiType.HTML);
+            case BOTH: cardinalConfigurationParameters.setUiType(CardinalUiType.BOTH);
```

Comment:
Not a huge concern since the values are distinct, but I just noticed these have always fallen through since we don't use break statements.

Reply:
Good catch! I'll be converting this file to Kotlin in another PR. This issue should be resolved by converting to a when statment.